test/include/seqan3/test/performance/naive_minimiser_hash.hpp

```diff
@@ -65,10 +67,18 @@ struct naive_minimiser_hash_fn
        if (shape.size() > window_size)
            throw std::invalid_argument{"The size of the shape cannot be greater than the window size."};
 
-       return std::forward<urng_t>(urange) | seqan3::views::kmer_hash(shape)
-                                           | std::views::transform([seed] (uint64_t i) { return i ^ seed; })
-                                           | ranges::view::sliding(window_size - shape.size() + 1)
-                                           | std::views::transform([] (auto const in)
+       auto forward = std::forward<urng_t>(urange) | seqan3::views::kmer_hash(shape)
+                                                   | std::views::transform([seed] (uint64_t i) { return i ^ seed; });
+       auto reverse = std::forward<urng_t>(urange) | seqan3::views::complement
+                                                   | std::views::reverse
+                                                   | seqan3::views::kmer_hash(shape)
+                                                   | std::views::transform([seed] (uint64_t i) { return i ^ seed; })
+                                                   | std::views::reverse;
+       auto both = seqan3::views::zip(forward, reverse)
+                   | std::views::transform( [ ] (auto i) {return std::min(std::get<0>(i), std::get<1>(i));});
```

Comment:
So far I know, for views we align with the | with the equal sign
```suggestion
       auto both = seqan3::views::zip(forward, reverse)
                 | std::views::transform( [ ] (auto i) {return std::min(std::get<0>(i), std::get<1>(i));});
```